packages/cli/src/cli.ts

```diff
@@ -32,7 +32,10 @@ export function getCli({ add, generate, remove, update }: Commands) {
     .description("Add a new chain spec to the list")
     .argument("<key>", "Key identifier for the chain spec")
     .addOption(config)
-    .option("-f, --file <filename>", "Source from metadata encoded file")
+    .option(
+      "-f, --file <filename>",
+      "Source from metadata encoded file or runtime wasm",
+    )
```

Comment:
I would put it as a separate option, `--wasm`. This will make it way easier internally on the add command as well.

Reply:
Sounds great. At first thought I thought as well about that, but I inclined towards a simplest approach (in terms of public options) in the end.
Let's change it 👍🏻